praetorian_cli/sdk/chariot.py

```
@@ -66,17 +66,23 @@ def __init__(self, keychain: Keychain, proxy: str=''):
     def _make_request(self, method: str, url: str, **kwargs) -> requests.Response:
         """
-        Centralized method to make HTTP requests with proxy configuration.
-        Automatically applies proxy settings and verify=False if proxy is defined.
+        Centralized method to make HTTP requests to the Chariot API with all global headers/parameters set.
+        """
+        
+        self.add_beta_url_param(kwargs)
+
+        return self.request(method, url, self.keychain.headers(), **kwargs)
+    
+    def request(self, method: str, url: str, headers: dict = None, **kwargs) -> requests.Response:
```

Ruff (0.14.1)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)